lib/clusterable/ocn_resolution.rb

```diff
@@ -18,19 +19,50 @@ class OCNResolution
     #    scope :for_cluster, lambda { |_cluster|
     #      where(:$in => ocns)
     #    }
+    #
+    def self.table
+      Services[:holdings_db][:oclc_concordance]
+    end
+
+    def self.with_ocns(ocns)
+      return to_enum(__method__, ocns) unless block_given?
+      dataset = table.where(oclc: ocns).or(canonical: ocns)
+
+      dataset.each do |row|
```

Comment:
here's a chance to rename either the accessors `:deprecated, :resolved` or the table columns `oclc, canonical` so that they are the same, to bring harmony and reduce potential for confusion

Reply:
I don't love `oclc` and `canonical`; the old (pre-MongoDB) print holdings system called it `variant` and `canonical`, which I like better than `oclc` and `canonical`, but I'm not sure whether I prefer ` 

I think the only thing that relies on this table is:

https://github.com/hathitrust/hathitrust_catalog_indexer/blob/bb84f32c03c56108422363798df76ed996066750/lib/ht_traject/oclc_resolution.rb#L7

so that should be easy enough to change.

The table is also in db_image so we'd need to rename the columns there, but nothing other than catalog indexer is using it.



That's all limited enough impact that I think it's worth doing.

Reply:
I think we can do this as part of https://hathitrust.atlassian.net/browse/DEV-1448.